RELEASE.md

```diff
@@ -15,7 +15,6 @@
 * Added `astro-iris` as alias for `astro-airlow-iris`, so that old tutorials can still be followed.
 * Added details about [Kedro's Technical Steering Committee and governance model](https://kedro.readthedocs.io/en/0.17.7/14_contribution/technical_steering_committee.html).
 
-## Minor breaking changes to the API
```

Comment:
Optional: I know this is not your change, but could we improve the links on lines 9 & 10 so that it points to actual links we'd have on 0.17.7 readthedocs? Like in line 16.